pkg/pdutil/pd.go

```diff
@@ -257,19 +276,17 @@ func (p *PdController) getRegionCountWith(
 
 // PauseSchedulers remove pd scheduler temporarily.
 func (p *PdController) PauseSchedulers(ctx context.Context, schedulers []string) ([]string, error) {
-	return p.pauseSchedulersWith(ctx, schedulers, pdRequest)
+	return p.pauseSchedulersAndConfigWith(ctx, schedulers, nil, PDRequest)
 }
```

Comment:
can we remove this function? looks problematic if accidentally called instead of RemoveSchedulers  (and no one else uses PauseSchedulers anyway).

Reply:
ok, I'll remove it,and also `pauseSchedulersAndConfig`